src/trackForMutations.js

```diff
@@ -1,26 +1,36 @@
-export default function trackForMutations(isImmutable, obj) {
-  const trackedProperties = trackProperties(isImmutable, obj);
+export default function trackForMutations(isImmutable, ignore, obj) {
```

Comment:
good point, and agree not really necessary 👍  I looked at it a bit and not really sure I even like that any better, so I won't make this change.

As-is, it's kinda nice that the `index` file is the only one concerned with destructuring the "options" instead of having to do that in two places.  But maybe the next time a new option is implemented, the implementor can reassess :)